app/package.json

```diff
@@ -10,27 +10,25 @@
     "format:js": "npm run eslint -- --fix",
     "format:scss": "npm run stylelint -- --fix",
     "format": "npm run format:js && npm run format:scss && prettier --write */**/*.md",
-    "dev": "webpack-dev-server --port 3000",
-    "build": "webpack -p --env.production",
-    "storybook": "start-storybook -p 9001 -c .storybook",
-    "storybook:build": "build-storybook -c .storybook -o .storybook_app",
+    "dev": "NODE_ENV=development webpack-dev-server --port 3000",
+    "build": "webpack -p",
     "test": "jest --notify",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "build:lang:en": "webpack --display none --env.dev && babel localization/translator.js | node",
```

Comment:
Also please update this scripts with proper `--env` options

Reply:
production mode used by default.